src/Backend/OnboardingInsuranceAPI/Areas/User/UserController.cs

```diff
@@ -0,0 +1,61 @@
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using OnboardingInsuranceAPI.Extensions;
+using System.Net;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace OnboardingInsuranceAPI.Areas.User;
+
+public class UserController
+{
+    private readonly GetUser _handlerGet;
+    private readonly RegisterUser _handlerPost;
+    private readonly UpdateUser _handlerPut;
+    private readonly ILogger<UserController> _log;
+
+    public UserController(GetUser handlerGet, RegisterUser handlerPost , UpdateUser handlerPut, ILogger<UserController> log)
+    {
+        _handlerGet = handlerGet;
+        _handlerPut = handlerPut;
+        _handlerPost = handlerPost;
```

Comment:
You could probably name it better. Usually just the name of the class is okay.

Reply:
Yeah I'll do